vollyball_scheduling.py

```diff
@@ -4,11 +4,18 @@
 Andrew Tatarek built in Microsoft Excel. The output of this file is a grid where each column is a set of games and
 rows repeat: "team1, team2, referee" for each match.
 
-The code in this file was written jointly by Micheal Howlett and David Howlett.
+Tries to minimize score =
+    Number of games where the referee is from a different group * 0.1
+    + (max(games played per team) - min(games played per team)) * 5
+    + sum(max(games played per team) - min(games played per team) for each group) * 2
+    + sum(max(games played or refereed per team) - min(games played or refereed per team) for each group)
+
+The code in this file was written jointly by Michael Howlett and David Howlett.
```

Comment:
Please add yourself to the list of authors

Reply:
Done